lib/active_merchant/billing/gateways/kushki.rb

```diff
@@ -185,7 +186,8 @@ def add_contact_details(post, contact_details_options)
       end
 
       def add_full_response(post, options)
-        post[:fullResponse] = options[:full_response].to_s.casecmp('true').zero? if options[:full_response]
+        # this is the only currently accepted value for this field, previously it was 'true'
+        post[:fullResponse] = 'v2' unless options[:full_response].nil? || !options[:full_response]
```

Comment:
Just to double-check: the idea here is still to send this field only if the request explicitly includes it in options, right?

Reply:
Yes, I had it written a different way and rubocop was being weird. This kinda looks weird to me so if you have a better idea that I'm too tired to see staring me in the face, fire away!

Reply:
Maybe `.blank?`

https://blog.arkency.com/2017/07/nil-empty-blank-ruby-rails-difference/#_code_blank___code_

Reply:
Used `.blank?` and also allowed for if someone passed it as false. But since we stringify everything and won't be explicitly changing it to a boolean like we do for others left 'false' as a string in the check.